src/sentry/services/hybrid_cloud/integration/impl.py

```diff
@@ -343,7 +342,7 @@ def send_incident_alert_notification(
         incident_id: int,
         organization: RpcOrganizationSummary,
         new_status: int,
-        incident_attachment: Mapping[str, str],
+        incident_attachment: Mapping[str, Any],
```

Comment:
Why loosen the type here?

Reply:
Just aligning with reality. The values returned by `sentry.rules.actions.notify_event_service.build_incident_attachment` have always had a nested dict as one of the values.

I've got a TODO in `service.py` to change the RPC interface -- an `Any` will be a no-go if we want to publish the schema.